plugins/org.jkiss.dbeaver.data.office/OSGI-INF/l10n/bundle_ko.properties

```diff
@@ -17,6 +17,8 @@ dataTransfer.processor.xlsx.property.boolStringFalse.description=FALSE(Boolean)
 dataTransfer.processor.xlsx.property.exportSql.name=SQL \uCD94\uCD9C
 dataTransfer.processor.xlsx.property.exportSql.description=SQL \uACB0\uACFC\uB97C \uB450\uBC88 \uC9F8 \uC2DC\uD2B8\uB85C \uCD94\uCD9C
 dataTransfer.processor.xlsx.property.extension.label = \uD30C\uC77C \uD655\uC7A5\uC790
+dataTransfer.processor.xlsx.property.headerFontName.name=\u000d\u000a\ud5e4\ub354\u0020\uae00\uaf34\u0020\uc774\ub984
+dataTransfer.processor.xlsx.property.headerFontName.description=\uccab \ubc88\uc9f8 \ud589 \uae00\uaf34 \uc774\ub984
```

Comment:
Until you're sure that you know that language, such translations are useless and tend to contain grammatical errors.